src/containers/App.js

```diff
@@ -1,22 +1,12 @@
 import React, { Component } from 'react';
-import { Redirect } from "react-router-dom"
 import { connect } from 'react-redux';
-import { Header } from '../components/Header';
 import { Main } from '../components/Main';
 import './App.css'
 
 class App extends Component {
 
   render() {
-
-
-    return (
-      <React.Fragment>
-
-          <Main />
-
-          </React.Fragment>
-    )
+    return <Main />
```

Comment:
Will you add to App more components? I asked you because you create App component and use only Main inside. Maybe you should use Main component instead of the App component.